expo/expo
# [docs] Update xdl schemas fixing broken links

# Why

I made some changes to get rid of relative and broken URLs within the schemas.

# How

- `$ yarn schema-sync 36`
- `$ yarn schema-sync 37`
- `$ yarn schema-sync 38`
- `$ yarn schema-sync 39`
- `$ yarn schema-sync 40`

# Test Plan

- Run locally
- Check `/versions/vXX.0.0/config/app/` for versions `36..40`
